<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDFDialog</class>
 <widget class="QDialog" name="RDFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="spinBox_NTrees">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>10</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_MaxDepth">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>33</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_NTrees">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Trees</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MaxDepth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>34</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Depth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProbX_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>59</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Probe Distance X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProbY_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>81</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Probe Distance Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_probX">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>57</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_probY">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>80</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Training</string>
   </property>
   <widget class="QPushButton" name="loadTrainData_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_train">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Test</string>
   </property>
   <widget class="QLabel" name="label_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadTestData_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
